docs/proposals/edgemesh-design.md

```diff
@@ -76,6 +76,113 @@ Add two endpoints definition:
 |ServiceBus|Target|Request Forward|
 |Router|Source|Resutst Forward, Service Discovery, Load Balance|
 
+<img src="../images/proposals/router-design.png">
+
+* R1: EdgeController watches on k8s apiserver for Services/Endpoints.
+* R2: EC get the service definition and create router rule definition and save in DB(sqlite).
+* R3: Router gets the rule definition from DB and initializes the listeners/handlers for source and target.
+* R4: Users/Applications create a request.
+* R5: Router receives the request and forwards it to the target based on rule.
+
+In Alpha1.0 only servicebus and eventbus will be supported.
```

Comment:
I think we need to give some details on what is eventbus.  This component doesn't appear anywhere in our architecture diagram yet .

Reply:
Done, moved it under provider section so that eventbus presence is clear.